SourceX/DiabloUI/selhero.cpp

```diff
@@ -358,7 +358,7 @@ static bool shouldPrefillHeroName()
 void selhero_ClassSelector_Select(int value)
 {
 	int hClass = vecSelHeroDlgItems[value]->m_value;
-	if (gbSpawned && (hClass == PC_ROGUE || hClass == PC_SORCERER || hClass == PC_BARD)) {
+	if (gbSpawned && hClass != PC_WARRIOR) {

```

Comment:
This is just an inconsistency I noticed

Reply:
What's the inconsistency?

Reply:
`PC_BARBARIAN` was allowed

Reply:
Oh, the barbarian, good catch :)

Reply:
Ha ha, just realized this wasn't an oversight :D